README.md

```diff
@@ -36,13 +36,17 @@ Once the application environment has been configured and dependencies have been
 ```shell
 npm run serverless:deploy
 
-App deployed to: https://twilio-live-interactive-video-1234-5678-dev.twil.io
+App deployed to: https://twilio-live-interactive-video-1234-5678-dev.twil.io?passcode=12345612345678
 Passcode: 123 456 1234 5678
 This URL is for demo purposes only. It will expire on Tue Oct 19 2021 14:58:20 GMT-0600 (Mountain Daylight Time)
 ```
 
 If you make any changes to this application, you can run `npm run serverless:remove` followed by `npm run serverless:deploy` to deploy the new changes to your application.
 
+**NOTE:** The Twilio Function that provides access tokens via a passcode should *NOT* be used in a production environment. This token server supports seamlessly getting started with the collaboration app, and while convenient, the passcode is not secure enough for production environments. You should use an authentication provider to securely provide access tokens to your client applications. You can find more information about Programmable Video access tokens [in this tutorial](https://www.twilio.com/docs/video/tutorials/user-identity-access-tokens).
+
+The passcode will expire after one week. To generate a new passcode, run `npm run serverless:remove` followed by `npm run serverless:deploy`.
```

Comment:
`npm run serverless:deploy -- --override` also gets a new passcode and expiration date, while keeping the same serverless domain name as before. Not sure if it's worth mentioning, but otherwise this looks good.